cql3/sets.cc

```diff
@@ -121,7 +121,7 @@ sets::literal::to_string() const {
     return "{" + join(", ", _elements) + "}";
 }
 
-sets::value
+shared_ptr<sets::value>
 sets::value::from_serialized(const fragmented_temporary_buffer::view& val, set_type type, cql_serialization_format sf) {
     try {
```

Comment:
We're trying to move away from extensive use of shared_ptr, this is making it harder.